### PR TITLE
Updated README.md file

### PR DESCRIPTION
**Pull request by coding-geek21**
issue : #2

### CODE REVIEW

**Comment by SurendarSingh** [comment]
The command to run the website on localhost:3000 is
Run this command to start the project
node app.js

File: README.md [hunk 1]
```diff
@@ -1,3 +1,133 @@
...
+```bash
+git status
+```
+:four: Check for your remote
+```bash
+git remote -v
+```
+:five: Push changes to remote
+```bash
+git push origin '<your_branch_name>'
```
Comment (alphaX86) [inline README.md:128]:
```suggestion
git push -u origin '<your_branch_name>'
```

Comment (alphaX86) [reply README.md:128]: Also, mention a point like "during this stage, a sign in pop-up will appear in GitHub requesting for OAuth. Sign in using OAuth browser session or create a Personal Access Token to login.

Comment (alphaX86) [reply README.md:128]: Coz passwords are not accepted since Aug 2021 by GitHub.

**Review by alphaX86 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (coding-geek21) [reply README.md:128]: Yeah I will make changes

**Review by alphaX86 — APPROVED** [review]
LGTM 👍 good to merge

**Comment by alphaX86** [comment]
Thanks @coding-geek21

**Pull request merged**